CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## Forest unreleased
+Notable updates:
+
+### Added
+
+### Changed
+* [libp2p] Ban peers with duration: [#2396](https://github.com/ChainSafe/forest/pull/2396)
```

Comment:
```suggestion
* [libp2p] Ban peers with duration. Banned peers are automatically unbanned after a period of 24h.: [#2396](https://github.com/ChainSafe/forest/pull/2396)
```

Reply:
Where does the 24h period come from?

Reply:
It's 1h atm. Would u suggest changing the code to 24h or the doc to 1h?

Reply:
@hanabi1224 @elmattic Hm, I must've dreamt about it. In general, I'm for a 24h ban.

Reply:
Yes, 24h ban seems good.